src/Spatial.Roslyn.Tests/ApiCompare/Reflector.cs

```diff
@@ -11,11 +10,7 @@ public class Reflector
         private Assembly legacy;
         private Assembly current;
 
-        private string oldAssemblyPath;
-        private string newAssemblyPath;
```

Comment:
These are unused but the idea was to keep the paths, so that the assembly lookup could search their locations too - currently it only searches the current path - which is not really ideal

Reply:
We add them back later when they are used?

Reply:
sure